index.html

```diff
@@ -262,45 +262,63 @@ <h3><img src="images/Rectangle 55.png" alt="" />Language</h3>
         </p>
         <form action="https://formspree.io/f/xvoneaew" method="post">
           <div class="details">
-            <input
+            <label for="firstName">
+              <input
+            id="firstName"
               name="firstName"
               required
               class="hidden"
               name="firstName"
               type="text"
               placeholder="First Name"
             />
+            </label>
+           <label for="lastName">
             <input
-              class="hidden"
-              name="lastName"
-              required
-              type="text"
-              placeholder="Last Name"
-            /><br />
+            class="hidden"
+            name="lastName"
+            id="lastName"
+            required
+            type="text"
+            placeholder="Last Name"
+          />
+           </label>
+           
+            <br />
+          <label for="fullName">
             <input
-              type="text"
-              class="shown"
-              name="FullName"
-              required
-              maxlength="30"
-              placeholder="Full name"
-            />
+            type="text"
+            class="shown"
+            name="FullName"
+            id="fullName"
+            required
+            maxlength="30"
+            placeholder="Full name"
+          />
+          </label>
+          <label for="email">
             <input
-              type="email"
-              name="email"
-              placeholder="Email Address"
-              required
-            /><br />
+            type="email"
+            name="email"
+            id="email"
+            placeholder="Email Address"
+            required
+          />
+          </label>
+          <br />
+          <label for="textArea">
             <textarea
-              name="message"
-              id=""
-              cols="30"
-              rows="10"
-              placeholder="Write me something..."
-              maxlength="500"
-              required
-            ></textarea
-            ><br />
+            name="message"
+            id="textArea"
+            cols="30"
+            rows="10"
+            placeholder="Write me something..."
+            maxlength="500"
+            required
+          ></textarea
+          >
+          </label>
+          <br />
             <button class="change" type="submit">Get in touch</button>
```

Comment:
👍 Great job on the visual appearance of your button! However, it's important to ensure that the button meets the Web Content Accessibility Guidelines (WCAG) for contrast ratio. As recommended by WAI, web pages should have a minimum contrast by default, with a contrast ratio of at least 4.5:1 for normal-sized text. Unfortunately, your button's color combination fails to meet this requirement. To improve accessibility, please consider updating the button color to provide sufficient contrast with the background. You can use online contrast checkers or color contrast accessibility tools to verify if the new color combination meets the required standards. 🎨 Keep up the good work!💪
![image](https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/55840999/e27e9c0c-b6d5-4245-92ef-765a9b2f0a2e)